index.js

```diff
@@ -10,9 +10,14 @@ const { parse, generatePreviewHead } = require('./util');
 module.exports = {
   name: require('./package').name,
 
+  configKey: 'ember-cli-storybook',
```

Comment:
This can be removed from this PR :)

Reply:
Why do we want to remove this?

Reply:
This was already handled in https://github.com/storybookjs/ember-cli-storybook/pull/54/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R13